desktop/apps/artwork/components/artists/templates/biography.jade

```diff
@@ -14,7 +14,10 @@
   .artwork-artist__content__short-bio
     p= artist.bio.replace('born', 'b.')
 
-if artist.exhibition_highlights && artist.exhibition_highlights.length <= 15
-  include ./exhibition
-else if artist.exhibition_highlights && artist.exhibition_highlights.length > 15
-  include ./highlights
+div(id="market-insights-container-" + artist._id)
+
+unless isAdmin
```

Comment:
Here is the corresponding change to making that tab always show up: for admins we don't even show a 'preview' inline.